packages/integration-karma/test/api/buildCustomElementConstructor/index.spec.js

```diff
@@ -9,7 +9,7 @@ const SUPPORTS_CUSTOM_ELEMENTS = !process.env.COMPAT && 'customElements' in wind
 
 function testInvalidOptions(type, obj) {
```

Comment:
This test can be removed because it was designed to make sure that `buildCustomElementConstructor` do throw the right error when being invoked with exotic parameters. No need to test it against `getComponentDef`, it should have been tested somewhere else.